ykview/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ykview"
 version = "0.1.0"
-authors = ["bjorn3 <bjorn3@users.noreply.github.com>"]
+authors = ["Edd Barrett <vext01@gmail.com>"]
```

Comment:
???

```suggestion
authors = ["The Yorick Developers"]
```

Reply:
Oops! That was a copy-paste error! I assure you I wasn't trying to take credit!